websockify/websocketproxy.py

```diff
@@ -392,6 +406,12 @@ def websockify_init():
             help="disallow non-encrypted client connections")
     parser.add_option("--ssl-target", action="store_true",
             help="connect to SSL target as SSL client")
+    parser.add_option("--verify-client", action="store_true",
+            help="require encrypted client to present a valid certificate")
+    parser.add_option("--cafile", metavar="FILE",
+            help="file of concatenated certificates of authorities trusted "
+            "for validating clients (only effective with --verify-client). "
+            "If omitted, system default list of CAs is used.")
```

Comment:
This isn't completely true, is it? The code suggests that system CAs will always be used.

Reply:
Indeed. I should move the call to `context.set_default_verify_paths()` into the `else` alternative.